deps/build.jl

```diff
@@ -26,11 +32,19 @@ if install_gap
     ## TODO: We currently use the GAP master branch.
     ##       Once all issues of using GAP with the julia
     ##       GC are resolved, we switch to a stable version.
-    run(`git clone https://github.com/gap-system/gap`)
+    run(`git clone --depth=1 https://github.com/gap-system/gap`)
     cd("gap")
     run(`./autogen.sh`)
     run(`./configure --with-gc=julia --with-julia=$(julia_binary)`)
     run(`make -j$(Sys.CPU_THREADS)`)
+    ## Hack to make GAPTypes available in the global package environment:
+    ## For a moment, we set the Julia load path to the actual depot, i.e.,
+    ## the env GAP.jl is installed to, to install GAPTypes.jl globally
+    julia_load_path_save = ENV["JULIA_LOAD_PATH"]
```

Comment:
Is `ENV["JULIA_LOAD_PATH"]` guaranteed (by documentation / specification) to be set at this point? If it is not, then this line could throw an exception.

Reply:
Since this build script is (or better should, nobody is keeping you from shooting yourself in the foot) only executed by the package manager, yes, I am sure this is always set.